CourseApp/Program.cs

```diff
@@ -4,10 +4,64 @@ namespace CourseApp
 {
     public class Program
     {
+        public static double Calc(double a, double b, double x)
+        {
+            var numerator = Math.Log(Math.Abs(Math.Pow(b, 2) - Math.Pow(x, 2)), a);
+            var denominator = Math.Pow(Math.Abs(Math.Pow(x, 2) - Math.Pow(a, 2)), 1 / 5.0);
```

Comment:
please check POW(-1, 1/5.0) - it should exist. Othervise please add something like this:  Math.Cbrt() (instead of 1/3.0)

```c
private static double CubeRoot(double x) {
    if (x < 0)
        return -Math.Pow(-x, 1d / 3d);
    else
        return Math.Pow(x, 1d / 3d);
```